packages/apps-routing/src/contracts.ts

```diff
@@ -23,13 +23,14 @@ export default function create (t: TFunction): Route {
   return {
     Component,
     display: {
-      needsAccounts: true,
+      needsAccounts: false,
       needsApi: [
         'tx.contracts.instantiateWithCode'
       ],
       needsApiCheck
     },
     group: 'developer',
+    href: 'http://contracts-ui.substrate.io/',
```

Comment:
Why not `https`?

Reply:
Contracts UI is a generic page with multiple possible backends, kinda like polkadot.js.org. Would be nice to set target backend directly, something along the lines:
`https://contracts-ui.substrate.io/?rpc=wss://ws.test.azero.dev`
Of course, if possible, we would like to have:
`https://contracts-ui.substrate.io/?rpc=[WEBSOCKET WE ARE NOW CONNECTED TO]`

Reply:
Oh yeah, the second idea is very good :+1:

Reply:
Done